.travis.yml

```diff
@@ -12,10 +12,14 @@ env:
   - ENGINE=GEOS
 
 before_script:
+  - if [[ $ENGINE = PDO_PGSQL ]]; then psql -c 'create database geo_tests;' -U postgres; fi;
+  - if [[ $ENGINE = PDO_PGSQL ]]; then psql -c 'create database geo_tests_tmp;' -U postgres; fi;
   - if [[ $ENGINE = PDO_MYSQL ]]; then bash .travis.install-mysql.sh; fi;
+  - if [[ $ENGINE = PDO_MYSQL ]]; then mysql -e 'create database geo_tests;'; fi;
+  - if [[ $ENGINE = PDO_MYSQL ]]; then mysql -e 'create database geo_tests_tmp;'; fi;
   - if [[ $ENGINE = GEOS ]]; then bash .travis.install-geos.sh; fi;
   - if [[ $ENGINE = SQLite3 ]]; then bash .travis.install-sqlite3.sh; fi;
-  - composer install
+  - composer install --prefer-source
```

Comment:
Could you use [preferred-install](https://getcomposer.org/doc/04-schema.md#config) in composer.json instead? So that no-one is confused by missing classes when cloning the repo and running a bare `composer install`.